### ISSUE TITLE
black blocks

### ISSUE THREAD

**Issue by awsomemonkeyguy** [DPS2004/Beatblock-issues#189]
**Describe the bug**
images of blocks placed down dosent appear and is instead just black

**To Reproduce**
ok i will be honest? i dont know how / when this bug appears.

when it does appear though, it seems to be fixed until you click on one of them

**Expected behavior**
have the image of the block used be visible

**Screenshots**
how the bug looks like
![bug](https://github.com/user-attachments/assets/b54a868e-0300-4196-809f-5f27fd5a74d9)

rentering the game seems fine
![Skärmbild 2024-12-06 114953](https://github.com/user-attachments/assets/b73dde53-6956-4070-a2a6-6f6a66d8739e)
until you click on it (stops being black when you stop clicking on it)
![Skärmbild 2024-12-06 115135](https://github.com/user-attachments/assets/10fa1a31-1c25-4369-9ca2-ea014b6b3cb9)
or if you make a new one
![Skärmbild 2024-12-06 115244](https://github.com/user-attachments/assets/af8bd0d5-5719-4cee-b8dd-4296e2d75e2e)

**Additional context**
apparently its something to do with tap notes?
![Skärmbild 2024-12-06 114159](https://github.com/user-attachments/assets/1b46908c-2905-4633-8f33-1cd5a1e614cc)
for me i think its if you have too many blocks you it wont load image anymore, basicly a limit on how many you can have in a level but i could be wrong i have no idea tbh
btw from my testing this bug seems to work on everything except notes.

side note: if any writing mistakes srry i just want this bug to be fixed as soon as possible cause the only current solution is some lua thing (spoiler: i dont know lua)

**By submitting this, I am confirming that I have read the [README](https://github.com/DPS2004/Beatblock-issues/blob/main/README.md) and am using the latest version of Beatblock.**

**Comment by DPS2004** [comment]
If you have a level file that reliably reproduces the issue, please send it here.

**Comment by awsomemonkeyguy** [comment]
level im working on where this glitch has happend (this is an actual chart so ignore everything except things mentioned here)
[ultimatum.zip](https://github.com/user-attachments/files/18053484/ultimatum.zip)
once downloaded and in the game, go to editor mode then go to EXACT beat 272.124, place anything other then a note (for example decoration), go up to 272.125 (or higher dosent matter) and you the image will turn black (this *should* work i have tested this with other people too if it does not then let me know beccuse then its probably a pc specific bug)
**tl;dr** if you are beffore 272.124 all images in editor work, going past 272.124 makes it not work.

just so you know i have *no idea* why this happens as nothing in the chart happens there 😭

also not 100% sure if this is true as i havent tested this alot so take this with a grain of salt
 but if you place anything after 272.124 (example decoration) and then save and exit the chart and then and go back there in editor it will show as if its fine even if you go past 272.124 but if you click the block you placed down it will be dark. 
**if you want me to, i can add step by step image guide for this**

**Comment by DPS2004** [comment]
Fixed in dev, will be available in the next version.
